### ISSUE TITLE
Applications Disappeared from Bungie.net

### ISSUE THREAD

**Issue by adh1310** [Bungie-net/api#899]
@vthornheart-bng As requested, here is a new issue for my, well, issue.

To recap, all but one of my applications on Bungie.net disappeared from my profile page. I seem to able to still access them from a direct link but they are not listed.

Both of my accounts are linked to my account as seen here:

![Capture](https://user-images.githubusercontent.com/8731789/55521149-7bbd8800-564d-11e9-832d-b4a226786e96.PNG)

When I visit the Applications page, this is what I see:

![Capture3](https://user-images.githubusercontent.com/8731789/55521174-9db70a80-564d-11e9-9ecc-a0254d7e4ce4.PNG)

Which is one of the applications I've used in the past. My other applications though aren't listed. They are still working though, I can use the API key and refresh token so they weren't deleted.

I get this error in the console when loading the page:

![Capture](https://user-images.githubusercontent.com/8731789/55521329-6137de80-564e-11e9-9643-ad518d63dac6.PNG)

Here is the link to the application page of one of the applications I can no longer see in my profile but can still access by direct link: [https://www.bungie.net/en/Application/Detail/29492](https://www.bungie.net/en/Application/Detail/29492)

And here is my profile URL: [https://www.bungie.net/en/Profile/254/6138511/TheAndrata](https://www.bungie.net/en/Profile/254/6138511/TheAndrata)

Thanks a million and thanks so much for what you guys do!

**Comment by vthornheart-bng** [comment]
Sweet, thank you!  Let me  pass this over to one of the UI guys to take a peek at the client-side error message, and I'll go take a look at whether something is up with the server side for your account.

**Comment by vthornheart-bng** [comment]
Huh, sure enough it's still there and still linked to you.  I'll bet a dollar that the client side error you're seeing is just preventing the rest of your applications from being shown: but I passed it up the chain, we'll see what we can find out.  So at least you can take comfort in the fact that the application you mentioned is indeed still tied to you!

We'll hit you up here when we find out more.

**Comment by vthornheart-bng** [comment]
Odd question - on that page, can you hit CTRL + F5 to force a full refresh, and then see if they appear?  Long shot, but I figure worth a try with the nature of the JS error you got.

**Comment by vthornheart-bng** [comment]
And a second follow-up question if that doesn't work: do you have any other error messages in the console when you open it up?  Try leaving it open and refreshing - if there's any other errors, pass them our way!

**Comment by hellocontrol-bng** [comment]
More questions:
- Does this happen in every browser?
- Do you have an ad blocker running?

And something to try:
- Open your browser console
- Paste `viewModels.application().list()` into it and hit enter
- Do you see more than one item in the resulting array?

**Comment by adh1310** [comment]
It's working! I logged in while at work today and noticed that everything was working as it should, all applications were showing up. Just got home and checked on my home computer and everything was back to not working. I tried all of the suggestions above as well as well as clearing out all cookies and cache, signing out of Google Chrome, and then reinstalling it so it would start fresh. And that seemed to resolve it. Not sure what was causing it but the nuclear option fixed it so, yeah. Now just have to sign back in to everything!

Thanks for all of the help, I really appreciate it!
